cmd/kubermatic-installer/cmd_local.go

```diff
@@ -172,6 +174,7 @@ func localKind(logger *logrus.Logger, dir string) (ctrlruntimeclient.Client, con
 		logger.Fatalf("failed to initialize runtime config: %v", err)
 	}
 
+	ctrlruntimelog.SetLogger(logr.New(ctrlruntimelog.NullLogSink{}))
```

Comment:
not suppressing the controller-runtime logger emits verbose error, but for the local installer this logger is not important

```
[controller-runtime] log.SetLogger(...) was never called, logs will not be displayed:
goroutine 1 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x5e
sigs.k8s.io/controller-runtime/pkg/log.eventuallyFulfillRoot()
        sigs.k8s.io/controller-runtime@v0.15.0/pkg/log/log.go:59 +0xcd
sigs.k8s.io/controller-runtime/pkg/log.(*delegatingLogSink).WithName(0xc000743e40, {0x28d6564, 0x5})
        sigs.k8s.io/controller-runtime@v0.15.0/pkg/log/deleg.go:147 +0x45
github.com/go-logr/logr.Logger.WithName(...)
...
```

Reply:
Instead of introducing a new dependency, we have zapr already IIRC and you should be able to just use Zap.